testdata/course-languages/bash/test-submissions/not-implemented/assignment.sh

```diff
@@ -0,0 +1,3 @@
+function add() {
+    echo "-1"
+}
```

Comment:
Missing newline (not an additional newline, just the newline that all files should end with).
Same in the other test submission.